### DISCUSSION TITLE
Standardize JSON casing

### DISCUSSION THREAD

**Discussion by CannonLock** [PelicanPlatform/pelican#1734] [General]
Current api keys use different 4 casing standards in its json responses.

https://github.com/PelicanPlatform/pelican/blob/main/server_structs/director.go

kebab-case
PascalCase
camelCase
snake_case

It would be nice to have some pattern in our use. If that is too challenging due to the conflicting technology and standards colliding it would be great to at least remove all use of kebab-case as it is unwieldy to work with on the frontend.

**Comment by CannonLock** [comment]
Just talked to @jhiemstrawisc, and we will be moving forward with camelCase*!

**Reply by jhiemstrawisc** [reply]
I just found this old issue:
https://github.com/PelicanPlatform/pelican/issues/899

There it says that at one point we settled on camel case for Web UI API data. Does that mean JSON?

**Reply by CannonLock** [reply]
Ah good to know, yeah that does.

**Reply by CannonLock** [reply]
I redact the previous statement. We will be moving forward with camelCase!
